azurerm/helpers/authentication/config.go

```diff
@@ -73,21 +73,21 @@ func (c *Config) LoadTokensFromAzureCLI() error {
 	foundToken := false
 	if c.TenantID != "" {
 		// pull out the ClientID and the AccessToken from the Azure Access Token
-		tokensPath, err := cli.AccessTokensPath()
-		if err != nil {
-			return fmt.Errorf("Error loading the Tokens Path from the Azure CLI: %+v", err)
+		tokensPath, err2 := cli.AccessTokensPath()
```

Comment:
I think its fair to exclude the shadowing rule for `err` as its just so common, so you don't have to do this. I think using some other name for an error than `err` should be avoided if at all possible.